### PR TITLE
Manual test and subs2redux

### PR DESCRIPTION
**Pull request by Velenir**
- Truffle script for running Pools locally in ganache

- Connecting subscriptions to redux devtools in development

- Fixes to subscriptions updates out of order.

- Misc fixes.

Truffle script won't work until gnosis/util-contracts#17 is merged

### CODE REVIEW

**Comment by ghost** [comment]
Travis automatic deployment:
 https://pr4--dxmgnpoolreact.review.gnosisdev.com

**Comment by ghost** [comment]
Travis automatic deployment:
 https://pr4--dxmgnpoolreact.review.gnosisdev.com

File: src/subscriptions/index.js [hunk 1]
```diff
@@ -256,20 +256,72 @@ export default async function startSubscriptions() {
...
+  subscription.on("error", (err) => {
+    console.error(
+      "An error in newBlockHeaders WS subscription occurred - unsubscribing.",
+      err.message || err,
+    )
         subscription.unsubscribe()
     })
 
-    return () => subscription && subscription.unsubscribe()
+  const interval = setInterval(() => AccountSub.update(), 2000)
```
Comment (W3stside) [inline src/subscriptions/index.js]: I don't really like this as it defeats the purpose of the websockets and not pinging infura over and over. Can we put this conditionally for development/localhost?

**Review by W3stside — COMMENTED** [review]
Looks good - subscriptions badly needed some rework - they probably need to be further minimized and at the very least moved into different files

**Comment by ghost** [comment]
Travis automatic deployment:
 https://pr4--dxmgnpoolreact.review.gnosisdev.com

**Comment by ghost** [comment]
Travis automatic deployment:
 https://pr4--dxmgnpoolreact.review.gnosisdev.com

**Comment by ghost** [comment]
Travis automatic deployment:
 https://pr4--dxmgnpoolreact.review.gnosisdev.com

**Comment by ghost** [comment]
Travis automatic deployment:
 https://pr4--dxmgnpoolreact.review.gnosisdev.com

**Comment by ghost** [comment]
Travis automatic deployment:
 https://pr4--dxmgnpoolreact.review.gnosisdev.com

**Comment by ghost** [comment]
Travis automatic deployment:
 https://pr4--dxmgnpoolreact.review.gnosisdev.com

**Comment by W3stside** [comment]
merge

**Review by W3stside — APPROVED** [review]

**Pull request merged**